lib/http.js

```diff
@@ -62,3 +63,16 @@ module.exports = {
   get,
   request
 };
+
+Object.defineProperty(module.exports, 'maxHeaderSize', {
+  configurable: true,
```

Comment:
should this be `configurable: false, writeable: false`?

Reply:
`writable` is not supported on accessor descriptors.